packages/terra-alert/src/terra-dev-site/doc/example/LongTextExample.jsx

```diff
@@ -1,31 +1,34 @@
 import React, { useState } from 'react';
-import Button from 'terra-button';
-import Toggle from 'terra-toggle';
+import ShowHide from 'terra-show-hide';
 import Alert from 'terra-alert';
+import classNames from 'classnames';
+import classNamesBind from 'classnames/bind';
+
+import styles from './LongTextExample.module.scss';
+
+const cx = classNamesBind.bind(styles);
 
 const LongTextExample = () => {
-  const [showAll, setShowAll] = useState(false);
+  const [isOpen, setIsOpen] = useState(false);
+  const previewText = "Four score and seven years ago our fathers brought forth on this continent, a new nation, conceived in Liberty, and dedicated to the proposition that all men are created equal.";
 
   return (
     <Alert type="info" title="Gettysburg Address:">
-      <span>
-        Four score and seven years ago our fathers brought forth on this continent, a new nation, conceived in Liberty, and dedicated to the proposition that all men are created equal.
-        <Button
-          variant="de-emphasis"
-          onClick={() => {
-            setShowAll(!showAll);
-          }}
-          text={showAll ? 'Less Details' : 'More Details'}
-        />
-        <Toggle isOpen={showAll}>
-          <p>
-            Now we are engaged in a great civil war, testing whether that nation, or any nation so conceived and so dedicated, can long endure. We are met on a great battle-field of that war. We have come to dedicate a portion of that field, as a final resting place for those who here gave their lives that that nation might live. It is altogether fitting and proper that we should do this.
-          </p>
-          <p>
-            But, in a larger sense, we can not dedicate -- we can not consecrate -- we can not hallow -- this ground. The brave men, living and dead, who struggled here, have consecrated it, far above our poor power to add or detract. The world will little note, nor long remember what we say here, but it can never forget what they did here. It is for us the living, rather, to be dedicated here to the unfinished work which they who fought here have thus far so nobly advanced. It is rather for us to be here dedicated to the great task remaining before us -- that from these honored dead we take increased devotion to that cause for which they gave the last full measure of devotion -- that we here highly resolve that these dead shall not have died in vain -- that this nation, under God, shall have a new birth of freedom -- and that government of the people, by the people, for the people, shall not perish from the earth.
-          </p>
-        </Toggle>
-      </span>
+      <ShowHide
+        preview={previewText}
```

Comment:
Need to either re-include the `{previewText}` as the first item with the children/contents or can remove the preview prop and provide the non-hidden text as just a span ahead of the `<ShowHide... />` component so is visible both before and after the show/hide.